Portfolio.css

```diff
@@ -1,3 +1,173 @@
+* {
+    box-sizing: border-box;
+}
+
 Body {
     font-family: 'Quicksand', sans-serif;
+    font-size: 62.5%;
+    background-color: #fff;
```

Comment:
Inconsistent use of long and short hex values (look at lines 8 and 9) - use long generally as short values can't do every colour :-)

Reply:
done.